ecl/eclagent/start_eclagent

```diff
@@ -34,4 +34,4 @@ else
 fi
 
 export SENTINEL="eclagent.sentinel"
-nohup eclagent $* &
```

Comment:
Should this file be renamed start_hthor?

Reply:
It could be... depends whether anything relies on old name I guess.